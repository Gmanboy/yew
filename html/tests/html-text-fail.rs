use yew_html::test_html;

test_html! { |t1| "valid" "invalid" }
test_html! { |t2| <span>{ "valid" "invalid" }</span> }

// unsupported literals
test_html! { |t10| b'a' }
test_html! { |t11| b"str" }
test_html! { |t12| 1111111111111111111111111111111111111111111111111111111111111111111111111111 }

fn main() {}
