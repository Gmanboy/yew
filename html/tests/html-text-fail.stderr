error: unsupported type
 --> $DIR/html-text-fail.rs:9:20
  |
9 | test_html! { |t12| 1111111111111111111111111111111111111111111111111111111111111111111111111111 }
  |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: unsupported type
 --> $DIR/html-text-fail.rs:8:20
  |
8 | test_html! { |t11| b"str" }
  |                    ^^^^^^

error: unsupported type
 --> $DIR/html-text-fail.rs:7:20
  |
7 | test_html! { |t10| b'a' }
  |                    ^^^^

error: unexpected token
 --> $DIR/html-text-fail.rs:4:35
  |
4 | test_html! { |t2| <span>{ "valid" "invalid" }</span> }
  |                                   ^^^^^^^^^

error: only one root html element allowed
 --> $DIR/html-text-fail.rs:3:1
  |
3 | test_html! { |t1| "valid" "invalid" }
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)
